src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
```

Comment:
Reporter: PMD
Rule: EmptyIfStmt
Severity: WARN
File: /src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java L9
 

Empty Code http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/empty.html#EmptyIfStmt

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>